face/src/components/execution/stage.hbs

```diff
@@ -0,0 +1,19 @@
+{{#if name}}
```

Comment:
this partial is used only once. Would be better to inline this into `stages-block`